packages/prism/README.md

```diff
@@ -38,5 +38,17 @@ export default ({ children }) => (
 Then, all code blocks in MDX documents wrapped by Layout will be
 syntax highlighted.
 
+Please note that `@theme-ui/prism` uses [`prism-react-renderer`](https://github.com/FormidableLabs/prism-react-renderer), [which does not include all languages supported by Prism](https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js) by default. So, if you happen to use one of the missing languages, your code block simply won't show up as highlighted.
```

Comment:
```suggestion
Please note that `@theme-ui/prism` uses [`prism-react-renderer`](https://github.com/FormidableLabs/prism-react-renderer), [which does not include all languages supported in Prism](https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js) by default. So, if you happen to use one of the missing languages, your code block simply won't show up as highlighted.
```

@jxnblk I think that two `by`'s right next to each other sounds weird, what do you think?